### ISSUE TITLE
Upgrade all not working

### ISSUE THREAD

**Issue by marionf** [PrestaShop/PrestaShop#12044]
**Describe the bug**
Try to upgrade all your modules and you will get a 500

**To Reproduce**
1. Go to Modules > Modules manager > Updates
2. Click on Update all
3. See error 500 in the console
4. Only the first module has been updated

**Screenshots**
![capture d ecran_871](https://user-images.githubusercontent.com/13449658/50765576-9793d680-1276-11e9-9a14-128979bcf760.png)

**Additionnal information**
PrestaShop version: 1.7.5.0
PHP version: N/A

**Comment by marionf** [comment]
Related to #12011

**Comment by Quetzacoalt91** [comment]
Indeed, all module actions ran as bulk will throw an error 500. This is caused by concurrent use of the cache while it is being deleted.

**Comment by SharakPL** [comment]
I don't see any erros in my console while updating modules with _Upgrade all_ button, but it only updates 1 module each time. Even though it takes some time for each module it never updates all of them. I've tried this on 2 shops 1.7.5.0 just now: one installed localy and the other on the server. Both showed 5 updates available and on both I had to hit _Upgrade all_ button 5 times. I think the same happened in 1.7.4

![updateall](https://user-images.githubusercontent.com/5007456/50980032-121f5900-14f8-11e9-8a8f-49b6cabb9591.gif)
